integration-test/1218-poni-whitelist.py

```diff
@@ -0,0 +1,309 @@
+# https://www.openstreetmap.org/node/2207370738
+# 15/5230/12657
+# aeroway"="helipad"
+assert_matching_feature(
```

Comment:
Rather confusingly, this is called `assert_has_feature` even though its opposite is `assert_no_matching_feature`.